### DISCUSSION TITLE
Adding your own entities

### DISCUSSION THREAD

**Discussion by Luffyyy** [nextauthjs/next-auth#1750] [Help] [answered]
**Summary of proposed feature**
The ability to add entities from the config, not just edit existing ones.

**Purpose of proposed feature**
It solves having to create an additional connection to the database for other possible tables we'd want to create for the site. Some people might be interested in creating more tables for other purposes than just authentication.

**Detail about proposed feature**
An array value in the adapter config that would just add the TypeORM entities (or the other adapter, haven't used it) that are inside it. Perhaps calling it something like custom_entities or custom_models.

*Please indicate if you are willing and able to help implement the proposed feature.*
Not sure. I might try.

**Comment by albertpak** [comment, marked as answer]
check this out: https://next-auth.js.org/tutorials/typeorm-custom-models - you can use this to expand any of the existing models and add custom attributes
